src/SLCore/Common/Helpers/ConnectionIdHelper.cs

```diff
@@ -33,6 +35,28 @@ internal class ConnectionIdHelper : IConnectionIdHelper
         private const string SonarQubePrefix = "sq|";
         private static readonly Uri SonarCloudUri = new Uri("https://sonarcloud.io");
```

Comment:
C#9 target-typed null can be useful here (it's not useful ultimately everywhere, this is a place where it is)
```
private static readonly Uri SonarCloudUri = new("https://sonarcloud.io");
```